server/src/entities/album.entity.ts

```diff
@@ -53,7 +53,7 @@ export class AlbumEntity {
   albumThumbnailAssetId!: string | null;
 
   @ManyToMany(() => UserEntity)
-  @JoinTable()
+  @JoinTable({name: "album_permissions"})
```

Comment:
I don't think this is correct and it will break existing installation

Reply:
Yeah, I'm still checking